services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -462,6 +462,18 @@ async def update_project_without_checking_permissions(
             )
             return result.rowcount == 1
 
+    async def update_project_last_change_timestamp(
+        self, project_uuid: ProjectIDStr
+    ) -> bool:
+        async with self.engine.acquire() as conn:
+            result = await conn.execute(
+                # pylint: disable=no-value-for-parameter
+                projects.update()
+                .values(**{"last_change_date": now_str()})
+                .where(projects.c.uuid == project_uuid)
+            )
+            return result.rowcount == 1
```

Comment:
THOUGHT: this will **always** be true since you are updating to "now". Why do you think you need this return value?

Reply:
you are right, i dont need it will remove it